player/lua/stats.lua

```diff
@@ -576,8 +579,10 @@ local function append_display_sync(s)
         append_property(s, "vsync-jitter", {prefix="VSync Jitter:", suffix=o.prefix_sep .. jitter_graph})
     else
         -- Since no graph is needed we can print ratio/jitter on the same line and save some space
-        local vratio = append_property(s, "vsync-ratio", {prefix="VSync Ratio:"})
-        append_property(s, "vsync-jitter", {prefix="VSync Jitter:", nl="" or o.nl})
+        local vr = append_property(s, "vsync-ratio", {prefix="VSync Ratio:"})
+        append_property(s, "vsync-jitter", {prefix="VSync Jitter:",
+                            nl=vr and "" or o.nl,
+                            indent=vr and o.prefix_sep .. o.prefix_sep or nil})
```

Comment:
not sure about precedence
also you can drop `or nil`
```suggestion
                            indent=vr and (o.prefix_sep .. o.prefix_sep)})
```

Reply:
done